app/presenters/tree_node_builder.rb

```diff
@@ -30,8 +30,8 @@ def self.build(object, parent_id, options)
     builder.build
```

Comment:
parent_id is still used here (https://github.com/ManageIQ/manageiq/pull/6397/files#diff-a6464e3e4f5dba04beb9e34cb112365eR29), so killing it from the internals of TreeNodeBuilder entirely is a bit too aggressive.